### ISSUE TITLE
Some elements can't be accessed by Tab key on Safari&Firefox on mac

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt#3455]
**Repro steps:**
• Navigate to https://makecode.microbit.org/beta
• Use "Tab" key to access element on the page

**Expected Result:**
Every element should be accessed by Tab key
**Actual Result:**
**Issues 1:**
Many elements can't be accessed on Safari
**Issue 2:**
"Learn more" can't be accessed on Firefox on Mac 
![nov-10-2017 17-00-57](https://user-images.githubusercontent.com/23466737/32651171-48c3f7f2-c63b-11e7-86f3-ef730e1b4304.gif)

**Comment by microsoftsam** [comment]
Safari and Firefox on a Mac require certain configuration in order to capture tabbing on all elements. 
This isn't something we can fix on our side, you'll have to set the right accessibility options in order to get the expected results: 
see https://support.apple.com/en-us/HT204434

**Comment by microsoftsam** [comment]
See https://github.com/Microsoft/pxt/issues/2418 as well for more context
